### DISCUSSION TITLE
2007/11/23/%E3%80%8A%E5%A4%AA%E9%98%B3%E7%85%A7%E5%B8%B8%E5%8D%87%E8%B5%B7%E3%80%8B%E8%A7%82%E5%90%8E%E6%84%9F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#191] [Announcements]
# 2007/11/23/%E3%80%8A%E5%A4%AA%E9%98%B3%E7%85%A7%E5%B8%B8%E5%8D%87%E8%B5%B7%E3%80%8B%E8%A7%82%E5%90%8E%E6%84%9F

得益于中国反盗版的不利，我已经看过数不清的电影了。虽然我肯花时间去看的电影都是有一定好评的电影，但是，按照我对它们的喜爱程度，还是可以再把它们分为三类：少数电影我看了之后仍然留在电脑上，过后还会再欣赏几遍；大多数电影虽然也很优秀，但都不值得我重看；还有少一部分是被宣传给骗了，看上几眼觉得没意思，我就开始往后拖进度条，如果全片都没什么亮点，可能几分钟就浏览完全片了。

https://blog.qizhen.xyz/2007/11/23/%E3%80%8A%E5%A4%AA%E9%98%B3%E7%85%A7%E5%B8%B8%E5%8D%87%E8%B5%B7%E3%80%8B%E8%A7%82%E5%90%8E%E6%84%9F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Zhang
2008年05月3日 00:19 编辑
我服了你们了，我为你们感到自豪！

回复
fang
2007年11月26日 09:53 编辑
我还没看呢，还要去看么。。。

回复
奕瑋
2007年11月25日 14:19 编辑
看到阮主和妍妍的不同观点，反倒让我有看看的想法了。呵呵。

回复
Lillian
2007年11月25日 01:01 编辑
看到博主做出这样的评论真是很遗憾。姜文绝不是在简单的说一个很乏味的故事，他是在隐藏如果不隐藏。可能这部电影我们就无缘观看了。可能在若干年厚我们的思想真正解放的时候我们会理解这部影片，我们会佩服姜文。影史上很多著名影片都是这么早就的。

回复
杰
2007年11月24日 19:12 编辑
你比我幸运的多，呵呵。
至少你没花钱到电影院去。

回复
半夏
2007年11月24日 10:05 编辑
当时看完走出影院完全一头雾水，姜文这次是自信了些。

回复
永青
2007年11月23日 20:49 编辑
我看了二十分钟再也没看下去。。。

回复
